pkgs/development/python-modules/jedi-language-server/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchFromGitHub
+, docstring-to-markdown
+, jedi
+, pygls
+, pytestCheckHook
+, pyhamcrest
+, python-jsonrpc-server
+}:
+
+let
+  pname = "jedi-language-server";
+  version = "0.34.8";
+
+  # Tests are not included in the pypi tarball. Installing from the github src
+  # is not possible because there is no setup.py there - upstream uses `poetry
+  # publish` to publish their package to pypi.
```

Comment:
this can also be accomplished with `format = "pyproject";`

I would just use the github source.

Reply:
> this can also be accomplished with `format = "pyproject";`
> 
> I would just use the github source.

That worked, thanks!